src/lib.rs

```diff
@@ -104,6 +104,8 @@
 
 // Make sure all our public APIs have docs.
 #![warn(missing_docs)]
+// Allowing/denying lints
```

Comment:
Cloud be made more clear, like "reject unused qualifications"

Reply:
`reject unused qualifications` is a specific linter of clippy.

What about keep this line and add comment for each linter?

Reply:
Ok, swap this line with the following line introducing the linter, and then comment on the linter.

Reply:
Fixed, PTAL~